meeting_rooms/serializers.py

```diff
@@ -1,19 +1,42 @@
 from rest_framework import serializers
 
-from .models import MeetingRoom
+from .models import MeetingRoom, Reservation, User
+from meeting_rooms.utils.error_codes import Errors as errors
 
 
-class MeetingRoomSerializer(serializers.ModelSerializer):
-    room_number = serializers.CharField()
-    custom_name = serializers.CharField(required=False)
-    capacity = serializers.IntegerField()
+class UserSerializer(serializers.ModelSerializer):
+    class Meta:
+        model = User
+        fields = '__all__'
 
+
+class MeetingRoomSerializer(serializers.ModelSerializer):
     class Meta:
         model = MeetingRoom
         fields = '__all__'
 
     def validate(self, data):
         if data.get('capacity') <= 0:
-            message = 'Capacity of the room must be a positive integer'
-            raise serializers.ValidationError(message)
+            error = {'non_field_errors': [errors.CAPACITY_NOT_POSITIVE_INTEGER_ERROR]}
+            raise serializers.ValidationError(error)
```

Comment:
```suggestion
            raise serializers.ValidationError(errors.CAPACITY_NOT_POSITIVE_INTEGER_ERROR)
```
Correct me if I'm wrong, but this would still return the same error structure, wouldn't it?

Reply:
You are right, fixed this.